infrastructure/variables.tf

```diff
@@ -1,6 +1,6 @@
 variable "product" {
   type    = "string"
-  default = "sscs-track-your-appeal"
+  default = "track-your-appeal-notifications"
 }
```

Comment:
This is not correct. Please revert it back to original "sscs-track-your-appeal".